Reads QC, Metagenome Assembly, Metagenome Annotation 500

Same error as before.   cc @jbeezley 

I rebuilt and cleared existing docker volumes.  

```
curl -X POST "http://localhost:8080/api/reads_qc/search?limit=100" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{\"conditions\":[]}"
```

```
backend_1  | 172.21.0.4:49606 - "POST /api/reads_qc/search?limit=100 HTTP/1.0" 500
backend_1  | [2020-07-20 19:50:34 +0000] [31] [ERROR] Exception in ASGI application
backend_1  | Traceback (most recent call last):
backend_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 385, in run_asgi
backend_1  |     result = await app(self.scope, self.receive, self.send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__
backend_1  |     return await self.app(scope, receive, send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/fastapi/applications.py", line 181, in __call__
backend_1  |     await super().__call__(scope, receive, send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/applications.py", line 102, in __call__
backend_1  |     await self.middleware_stack(scope, receive, send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__
backend_1  |     raise exc from None
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__
backend_1  |     await self.app(scope, receive, _send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__
backend_1  |     raise exc from None
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__
backend_1  |     await self.app(scope, receive, sender)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 550, in __call__
backend_1  |     await route.handle(scope, receive, send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle
backend_1  |     await self.app(scope, receive, send)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 41, in app
backend_1  |     response = await func(request)
backend_1  |   File "/usr/local/lib/python3.8/site-packages/fastapi/routing.py", line 204, in app
backend_1  |     response_data = await serialize_response(
backend_1  |   File "/usr/local/lib/python3.8/site-packages/fastapi/routing.py", line 125, in serialize_response
backend_1  |     raise ValidationError(errors, field.type_)
backend_1  | pydantic.error_wrappers.ValidationError: 100 validation errors for ReadsQCSearchResponse

```

--- a/nmdc_server/api.py
+++ b/nmdc_server/api.py
@@ -344,6 +344,22 @@ async def get_metagenome_annotation(metagenome_annotation_id: str, db: Session =
     return db_metagenome_annotation
 
 
+# metagenome_annotation
+@router.post(
+    "/metaproteomic_analysis/search",
+    response_model=query.MetaproteomicAnalysisSearchResponse,
+    tags=["metaproteomic_analysis"],
+    name="Search for studies",
+    description="Faceted search of metaproteomic_analysis data.",
+)
+async def search_metaproteomic_analysis(
+    query: query.SearchQuery = query.SearchQuery(),
+    db: Session = Depends(get_db),
+    pagination: Pagination = Depends(),
+):
+    return pagination.response(crud.search_metaproteomic_analysis(db, query.conditions))
+
+
 @router.post(
     "/metaproteomic_analysis/facet",
     response_model=query.FacetResponse,

--- a/nmdc_server/schemas.py
+++ b/nmdc_server/schemas.py
@@ -215,7 +215,6 @@ class PipelineStepBase(BaseModel):
     ended_at_time: datetime
     execution_resource: str
     project_id: str
-    stats: Dict[str, Union[int, float]]
 
 
 class PipelineStep(PipelineStepBase):

